### PR TITLE
Configurable pipelines

### PR DESCRIPTION
**Pull request by matentzn**
@hrshdhgd Can you check out this branch, and try to run:

```
make all
```

I am _loving_ this project.

### CODE REVIEW

File: config/Makefile.j2 [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+#####################
+
+ALL_MAPPINGS={% for mapping in mapping_registry.mapping_set_references %}$(TMP_DIR)/{{ mapping.local_name }} {% endfor %}
+
+{% for mapping in mapping_registry.mapping_set_references %}
+$(TMP_DIR)/{{ mapping.local_name }}: | $(TMP_DIR)/
+	wget {{ mapping.mirror_from|default(mapping.mapping_set_id) }} -O $@
+{% endfor %}
+
+$(BOOMER_INPUT_DIR)/%/combined.sssom.tsv $(BOOMER_INPUT_DIR)/%/prefix.yaml: $(ALL_MAPPINGS) | $(BOOMER_INPUT_DIR)/%/
```
Comment (matentzn) [inline config/Makefile.j2:70]: @hrshdhgd This make target needs fixing. Everything else I think works fine. Basically you can change all lines from `echo  "Harshad` to the end of the target. Basically you will have to rewrite gen_boomer_output in a way that it accepts the parameters i am passing in. Maybe add an Argparse or some such.

File: projects/mondo-icd10cm/boomer_input/exact_match_only/prefix.yaml [hunk 2]
```diff
@@ -1,8 +1,6 @@
-ICD10CM: http://purl.bioontology.org/ontology/ICD10CM/
-NCI: http://purl.obolibrary.org/obo/NCIT_
+ICD10CM: http://apps.who.int/classifications/icd10/browse/2010/en#/
```
Comment (matentzn) [inline projects/mondo-icd10cm/boomer_input/exact_match_only/prefix.yaml:1]: Why did this become ICD10CM again? Shouldnt it be whatever is used in icd10cm.owl?

Comment (hrshdhgd) [reply projects/mondo-icd10cm/boomer_input/exact_match_only/prefix.yaml]: That came from the [sssom.tsv file in mondo](https://github.com/monarch-initiative/mondo/blob/master/src/ontology/mappings/mondo_exactmatch_icd10cm.sssom.tsv)

Comment (hrshdhgd) [reply projects/mondo-icd10cm/boomer_input/exact_match_only/prefix.yaml]: I have fixed it in this project for now.

File: config/Makefile.j2 (hunk 1 above)
Comment (hrshdhgd) [reply config/Makefile.j2:73]: Done! But boomer takes a long time and doesn't go through will all mappings. I'm not sure if something is wrong. I had to forcibly stop.

**Review by hrshdhgd — APPROVED** [review]

**Pull request merged**